### PR TITLE
fix: render all slots inside a <template> vnode

### PR DESCRIPTION
**Pull request by eddyerburgh**
- Wrap all slot vnodes created from a string in a template vnode

Fixes https://github.com/vuejs/vue-test-utils/issues/967

### CODE REVIEW

File: packages/create-instance/create-slot-vnodes.js [hunk 1]
```diff
@@ -4,39 +4,32 @@ import { compileToFunctions } from 'vue-template-compiler'
...
-    vnode = vm.$createElement(slotValue)
-  }
-  if (vnode.data) {
-    vnode.data.slot = name
-  } else {
-    vnode.data = { slot: name }
+    return createVNodes(vm, slotValue, name)
   }
+  const vnode = vm.$createElement(slotValue)
+  ;(vnode.data || (vnode.data = {})).slot = name
```
Comment (lmiller1990) [inline packages/create-instance/create-slot-vnodes.js:32]: what does this line do?

Comment (eddyerburgh) [reply packages/create-instance/create-slot-vnodes.js:32]: It sets `vnode.data.slot` to equal the slot name. It's shorthand for:

```js
if(!vnode.data) {
  vnode.data = {}
}
vnode.data.slot = name
```

Comment (lmiller1990) [reply packages/create-instance/create-slot-vnodes.js:32]: I see! haven't seen this syntax before. I take it the preceeding `;` is required.

Comment (eddyerburgh) [reply packages/create-instance/create-slot-vnodes.js:32]: Yes, that's is one of the drawbacks of not including semi-colons, there are a few other cases where no semi-colons will cause errors.

Comment (lmiller1990) [reply packages/create-instance/create-slot-vnodes.js:32]: Fair enough.

Code looks good outside that, good job.

**Pull request merged**